### ISSUE TITLE
Hyrax Translation files not loaded when `hyrax:default_collection_types:create` task runs in Hyrax 2.1

### ISSUE THREAD

**Issue by mark-dce** [samvera/hyrax#3136]
### Descriptive summary

When I run the `hyrax:default_collection_types:create` task to create default collections, I get weird titles for the two collections that are created.

<img width="810" alt="screen shot 2018-06-28 at 3 50 09 pm" src="https://user-images.githubusercontent.com/298865/42060033-11c29140-7aeb-11e8-80d6-201317b707c6.png">

### Rationale

Users doing a 2.0 --> 2.1 upgrade following the upgrade instructions in the release notes are going to get collection names that make it look like internationalization is failing in the views, but there's actually bad title data being stored to the collection table in the database.

### Expected behavior
My new admin set gets named "Admin Set"

### Actual behavior
My new admin set gets named "translation missing: en.hyrax.collection_type.admin_set_title"

### Steps to reproduce the behavior

1. Run the `hyrax:default_collection_types:create` on an environment without any collections configured.

Here's an example run with a breakpoint added to the code

```
irb(main):002:0> Hyrax::CollectionType.count         
   (0.3ms)  SELECT COUNT(*) FROM "hyrax_collection_types"
=> 0
irb(main):003:0> exit
Marks-MacBook-Pro-2:laevigata mark$ bin/rails hyrax:default_collection_types:create

[15, 24] in /Users/mark/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/hyrax-2.1.0/app/models/hyrax/collection_type.rb
   15:     ADMIN_SET_MACHINE_ID = 'admin_set'.freeze
   16:     ADMIN_SET_DEFAULT_TITLE = I18n.t('hyrax.collection_type.admin_set_title').freeze
   17: 
   18:     byebug
   19: 
=> 20:     def title=(value)
   21:       super
   22:       assign_machine_id
   23:     end
   24: 
(byebug) ADMIN_SET_DEFAULT_TITLE
"translation missing: en.hyrax.collection_type.admin_set_title"
(byebug) I18n.t('hyrax.collection_type.admin_set_title').freeze
"translation missing: en.hyrax.collection_type.admin_set_title"
(byebug) # BUT !!!!
nil
(byebug) I18n.load_path += Dir['/Users/mark/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/hyrax-2.1.0/config/locales/hyrax.en.yml']
["/Users/mark/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/carrierwave-1.2.2/lib/carrierwave/locale/en.yml", "/Users/mark/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activesupport-5.1.6/lib/active_support/locale/en.yml", "/Users/mark/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activemodel-5.1.6/lib/active_model/locale/en.yml", "/Users/mark/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/activerecord-5.1.6/lib/active_record/locale/en.yml", "/Users/mark/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/actionview-5.1.6/lib/action_view/locale/en.yml", "/Users/mark/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/responders-2.4.0/lib/responders/locales/en.yml", "/Users/mark/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/active-fedora-11.5.2/lib/active_fedora/locale/en.yml", "/Users/mark/.rbenv/versions/2.4.2/lib/ruby/gems/2.4.0/gems/hyrax-2.1.0/config/locales/hyrax.en.yml"]
(byebug) I18n.t('hyrax.collection_type.admin_set_title').freeze
"Admin Set"
(byebug) 
```
### Related work

Link to related tickets or prior related work here.

**Comment by no-reply** [comment]
@elrayle do you have thoughts about this?

**Comment by RudyOnRails** [comment]
Oh rats, nevermind - I see that that the titles are incorrect only when you _don't_ have existing collections. I had made one in my 2.0 app before doing the upgrade demo...

**Comment by mark-dce** [comment]
At this point, a documentation warning seems fine - I'm guessing the number of folks impacted here is pretty low since this only occurs when there are no existing collections.  

My main question would be whether this has been addressed in later releases (2.3, 2.4, etc.) and whether the recommended upgrade path is 2.0-->2.1-->2.2-->2.3-->2.4 or just 2.0-->2.4.  If upgrading 2.0-->2.4 directly is the typical path, I'd consider this really low priority at this point.

**Comment by no-reply** [comment]
>  If upgrading 2.0-->2.4 directly is the typical path, I'd consider this really low priority at this point.

this is definitely the case. i don't think there's been any improvement on this front in later 2.x releases, and i think the impact is just as great for a 2.0 -> 2.5 upgrade as for 2.0 -> 2.1. but i agree that the impact seems pretty low.

i'm closing this, since we're edging toward the 3.0 release. if someone wants a fix for 2.x, feel free to bump.
